other/nightly/install.sh

```
@@ -711,84 +710,80 @@ curl -fsSL $CDN/docker-compose.prod.yml -o /data/coolify/source/docker-compose.p
 curl -fsSL $CDN/.env.production -o /data/coolify/source/.env.production
 curl -fsSL $CDN/upgrade.sh -o /data/coolify/source/upgrade.sh
 
-echo -e "6. Make backup of .env to .env-$DATE"
+echo -e "6. Setting up environment variable file"
 
-# Copy .env.example if .env does not exist
 if [ -f $ENV_FILE ]; then
+    # If .env exists, create backup
+    echo " - Creating backup of existing .env file to .env-$DATE"
     cp $ENV_FILE $ENV_FILE-$DATE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)